build/yamls/clickhouse-operator-install-bundle.yml

```diff
@@ -0,0 +1,3671 @@
+# Template Parameters:
```

Comment:
I don't know what his file is used for? If you want to add more docs related to network security, maybe we can involve them in a separate PR.

Reply:
Changed back.